src/fluid/tmunu.hpp

```diff
@@ -110,17 +108,17 @@ class StressEnergyTensorCon {
     SPACELOOP2(ii, jj) {
       const Real &bi = b_(ii + 1, std::forward<Args>(args)...);
       const Real &bj = b_(jj + 1, std::forward<Args>(args)...);
-      Bdotv += bi * vp[jj] / W * gcov[ii][jj];
+      Bdotv += bi * (vp[jj] / W) * gcov[ii][jj];
```

Comment:
Maybe rename `gcov` to `gamma_cov` or something? Thought I had found a bug for a second

Reply:
Done.